files/mender/device_type

```diff
@@ -0,0 +1 @@
+device_type=raspberrypi3
```

Comment:
Any ideas how to move that to devices directory?

Reply:
I made a note about it. I suggest we will discuss it later.